src/utils/styles.ts

```diff
@@ -0,0 +1,15 @@
+import { StyleSheet } from 'react-native';
+export const layoutStyles = StyleSheet.create({
+  ColumnJustifyStart: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'flex-start',
+  },
+  RowAlignCenter: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  noPaddings: { paddingLeft: 0, paddingRight: 0 },
+  margins: { marginRight: 2, marginTop: 4 },
```

Comment:
this is also too generic, and it's just 2 values, so I'd probably drop it in favor of simply adding these 2 values wherever they're needed

Reply:
Thanks. Commits 044c8f9 https://github.com/urbi-mobility/urbi-react-native/pull/233/commits/4a5bba8bb53655f4051b794c04e859f5dece4ac2